### ISSUE TITLE
关于代码参数的问题

### ISSUE THREAD

**Issue by QWERZPY** [PaddlePaddle/Research#240]
请问代码中参数设置需要调整吗，如果调用gpu的话会报错，调用cpu的话可以运行但没有输出epoch步骤也就是训练的步骤，直接进入了test

**Comment by sserdoubleh** [comment]
是不是 log_steps 设置得太大了，有训练日志可以发出来看看吗？

**Comment by QWERZPY** [comment]
好像没有找到log_steps，只看到了skip_steps,这个设置的是1000，训练日志见附件




如果调用gpu的话就会出现这个







在 2022-03-01 23:07:25，"sserdoubleh" ***@***.***> 写道：

是不是 log_steps 设置得太大了，有训练日志可以发出来看看吗？

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you authored the thread.Message ID: ***@***.***>
02/13/2022 18:54:31 - INFO - reader.data_reader - 10
02/13/2022 18:54:31 - INFO - model.gran_model - 10
ok
02/13/2022 18:54:31 - INFO - __main__ - 20
Namespace(attention_dropout_prob=0.2, batch_size=1024, checkpoints='./jf17k_out', do_predict=True, do_train=True, entity_soft_label=0.9, epoch=160, ground_truth_path='./data/jf17k/all.json', hidden_act='gelu', hidden_dropout_prob=0.2, hidden_size=256, init_checkpoint=None, init_pretraining_params=None, initializer_range=0.02, intermediate_size=512, learning_rate=0.0005, loss_scaling=1.0, lr_scheduler='linear_warmup_decay', max_arity=6, max_seq_len=11, num_attention_heads=4, num_edges=5, num_hidden_layers=12, num_iteration_per_drop_scope=1, num_relations=501, predict_file='./data/jf17k/test.json', relation_soft_label=1.0, skip_steps=1000, train_file='./data/jf17k/train.json', use_cuda=False, use_fast_executor=False, use_fp16=False, verbose=False, vocab_path='./data/jf17k/vocab.txt', vocab_size=29148, warmup_proportion=0.1, weight_decay=0.01, weight_sharing=True)
ok1
02/13/2022 18:54:31 - INFO - utils.args - -----------  Configuration Arguments -----------
02/13/2022 18:54:31 - INFO - utils.args - attention_dropout_prob: 0.2
02/13/2022 18:54:31 - INFO - utils.args - batch_size: 1024
02/13/2022 18:54:31 - INFO - utils.args - checkpoints: ./jf17k_out
02/13/2022 18:54:31 - INFO - utils.args - do_predict: True
02/13/2022 18:54:31 - INFO - utils.args - do_train: True
02/13/2022 18:54:31 - INFO - utils.args - entity_soft_label: 0.9
02/13/2022 18:54:31 - INFO - utils.args - epoch: 160
02/13/2022 18:54:31 - INFO - utils.args - ground_truth_path: ./data/jf17k/all.json
02/13/2022 18:54:31 - INFO - utils.args - hidden_act: gelu
02/13/2022 18:54:31 - INFO - utils.args - hidden_dropout_prob: 0.2
02/13/2022 18:54:31 - INFO - utils.args - hidden_size: 256
02/13/2022 18:54:31 - INFO - utils.args - init_checkpoint: None
02/13/2022 18:54:31 - INFO - utils.args - init_pretraining_params: None
02/13/2022 18:54:31 - INFO - utils.args - initializer_range: 0.02
02/13/2022 18:54:31 - INFO - utils.args - intermediate_size: 512
02/13/2022 18:54:31 - INFO - utils.args - learning_rate: 0.0005
02/13/2022 18:54:31 - INFO - utils.args - loss_scaling: 1.0
02/13/2022 18:54:31 - INFO - utils.args - lr_scheduler: linear_warmup_decay
02/13/2022 18:54:31 - INFO - utils.args - max_arity: 6
02/13/2022 18:54:31 - INFO - utils.args - max_seq_len: 11
02/13/2022 18:54:31 - INFO - utils.args - num_attention_heads: 4
02/13/2022 18:54:31 - INFO - utils.args - num_edges: 5
02/13/2022 18:54:31 - INFO - utils.args - num_hidden_layers: 12
02/13/2022 18:54:31 - INFO - utils.args - num_iteration_per_drop_scope: 1
02/13/2022 18:54:31 - INFO - utils.args - num_relations: 501
02/13/2022 18:54:31 - INFO - utils.args - predict_file: ./data/jf17k/test.json
02/13/2022 18:54:31 - INFO - utils.args - relation_soft_label: 1.0
02/13/2022 18:54:31 - INFO - utils.args - skip_steps: 1000
02/13/2022 18:54:31 - INFO - utils.args - train_file: ./data/jf17k/train.json
02/13/2022 18:54:31 - INFO - utils.args - use_cuda: False
02/13/2022 18:54:31 - INFO - utils.args - use_fast_executor: False
02/13/2022 18:54:31 - INFO - utils.args - use_fp16: False
02/13/2022 18:54:31 - INFO - utils.args - verbose: False
02/13/2022 18:54:31 - INFO - utils.args - vocab_path: ./data/jf17k/vocab.txt
02/13/2022 18:54:31 - INFO - utils.args - vocab_size: 29148
02/13/2022 18:54:31 - INFO - utils.args - warmup_proportion: 0.1
02/13/2022 18:54:31 - INFO - utils.args - weight_decay: 0.01
02/13/2022 18:54:31 - INFO - utils.args - weight_sharing: True
02/13/2022 18:54:31 - INFO - utils.args - ------------------------------------------------
02/13/2022 18:54:31 - INFO - __main__ - Device count: 64
02/13/2022 18:54:31 - INFO - __main__ - Num train instances: 313063
02/13/2022 18:54:31 - INFO - __main__ - Max train steps: 764
02/13/2022 18:54:31 - INFO - __main__ - Num warmup steps: 76
ok2
memory_optimize is deprecated. Use CompiledProgram and Executor
memory_optimize is deprecated. Use CompiledProgram and Executor
ok3
The CPU_NUM is not specified, you should set CPU_NUM in the environment variable list, i.e export CPU_NUM=1. CPU_NUM indicates that how many CPUPlace are used in the current task.
!!! The default number of CPUPlaces is 1.

I0213 18:56:11.992614 69700 parallel_executor.cc:329] The number of CPUPlace, which is used in ParallelExecutor, is 1. And the Program will be copied 1 copies
I0213 18:56:12.130934 69700 build_strategy.cc:340] SeqOnlyAllReduceOps:0, num_trainers:1
ok4
02/14/2022 05:22:02 - INFO - __main__ - Processing prediction steps: 0 examples: 0
02/14/2022 05:26:05 - INFO - __main__ - Processing prediction steps: 10 examples: 10240
02/14/2022 05:30:08 - INFO - __main__ - Processing prediction steps: 20 examples: 20480
02/14/2022 05:34:11 - INFO - __main__ - Processing prediction steps: 30 examples: 30720
02/14/2022 05:38:15 - INFO - __main__ - Processing prediction steps: 40 examples: 40960
02/14/2022 05:42:19 - INFO - __main__ - Processing prediction steps: 50 examples: 51200
02/14/2022 05:46:25 - INFO - __main__ - Processing prediction steps: 60 examples: 61440
02/14/2022 05:50:30 - INFO - __main__ - Processing prediction steps: 70 examples: 71680
02/14/2022 05:54:36 - INFO - __main__ - Processing prediction steps: 80 examples: 81920
02/14/2022 05:58:41 - INFO - __main__ - Processing prediction steps: 90 examples: 92160
02/14/2022 06:02:47 - INFO - __main__ - Processing prediction steps: 100 examples: 102400
02/14/2022 06:05:39 - INFO - __main__ - 
-------- Evaluation Performance --------
TASK	MRR	***@***.***	***@***.***	***@***.***	***@***.***
HEAD/TAIL	0.2458	0.1816	0.2655	0.3115	0.3701
PRIMARY_R	0.9784	0.9696	0.9855	0.9888	0.9924
ENTITY	0.2532	0.1897	0.2729	0.3169	0.3782
RELATION	0.9855	0.9797	0.9900	0.9923	0.9949
